### PR TITLE
build: add support of building with dkms

### PR DESCRIPTION
**Pull request by wentongwu**
Add support of building with dkms.

Signed-off-by: Wentong Wu <wentong.wu@intel.com>

### CODE REVIEW

**Comment by wentongwu** [comment]
@vicamo please review, Thanks!

**Comment by wentongwu** [comment]
@vicamo can you please take a look?

File: dkms.conf [hunk 1]
```diff
@@ -0,0 +1,38 @@
+PACKAGE_NAME="ivsc-driver"
+PACKAGE_VERSION="1.0.0"
+
+MAKE="make all"
+CLEAN="make clean"
```
Comment (vicamo) [inline dkms.conf]: Should always respect `$kernelver` and `$kernel_source_dir`:
```
MAKE="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir"
CLEAN="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir clean"
```

**Review by vicamo — CHANGES_REQUESTED** [review]

File: dkms.conf (hunk 1 above)
Comment (wentongwu) [reply dkms.conf]: where does kernelver and kernel_source_dir come from? can you please revisit? @vicamo

**Pull request merged**